OptimizelyDemoApp/AppDelegate.swift

```diff
@@ -73,7 +73,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         // ---- Create the Datafile Manager ----
         let datafileManager = OPTLYDatafileManagerDefault.init{(builder) in
             // builder!.datafileFetchInterval = TimeInterval(self.datafileManagerDownloadInterval)
-            builder!.datafileConfig = OPTLYDatafileConfig(projectId: nil, withSDKKey:self.projectId)!;
+            builder!.datafileConfig = OPTLYDatafileConfig(projectId: self.projectId, withSDKKey:nil)!;
```

Comment:
Is this change still valid? Shouldn't we be using SDK key by default?

Reply:
sure.  once we push it to prod and we can safely make that change, it will be done.